robot/include/robot.cpp

```diff
@@ -111,120 +147,51 @@ long double Autonomous::PathFinding::CalcBearing(Map::Coordinates source, Map::C
     return bearing;
 }
 
-void Autonomous::PathFinding::LineEquation(Map::Coordinates gps1, Map::Coordinates gps2, long double (&returnArr)[2]){
-    // slope value
-    returnArr[0] = (gps2.latitude-gps1.latitude)/(gps2.longitude-gps1.longitude);
-    // y-intercept value
-    returnArr[1] = (-returnArr[0])*gps1.latitude+gps1.longitude;
-}
-
-int Autonomous::PathFinding::FindPeak(vector<long double> height, bool reverse){
+void Autonomous::PathFinding::DrawMap(vector<Map::Coordinates> coordinateList, long double const SUBDIVISION, std::ofstream& file){
+    // Setup necessary storage and vars
+    vector<Map::Coordinates> *result = new vector<Map::Coordinates>{coordinateList[0]};
+    long double y_min = Map::Minimum(coordinateList, "long");
+    long double y_max = Map::Maximum(coordinateList, "long");
+    Map::Coordinates bottom = Map::Coordinates(coordinateList[0].latitude, y_min, 0);
+    Map::Coordinates top = Map::Coordinates(coordinateList[0].latitude, y_max, 0);
+    long double distance = Autonomous::PathFinding::CalcDistance(bottom,top);
+    int intervals = distance / SUBDIVISION;
```

Comment:
If I'm looking correctly, looks like increasing SUBDIVISION actually creates a larger area of coverage/distance between lines? Might be a little confusing